### DISCUSSION TITLE
nonebot2更新之后nonebot.adapters.Event.get_message()不会自动去掉on_command事件的消息的前面的命令部分,请问有啥方法代替么,谢谢

### DISCUSSION THREAD

**Discussion by Weltolk** [nonebot/discussions#78] [问答] [answered]
以前是能得,更新之后不能了,看了更新之后的文档,也没找到相应的代替方法,请问有啥方法代替么,谢谢

**Comment by yanyongyu** [comment, marked as answer]
https://v2.nonebot.dev/docs/tutorial/plugin/create-handler#commandarg

**Reply by Weltolk** [reply]
thanks,solved
另外说下,https://v2.nonebot.dev/docs/tutorial/plugin/create-handler#shellcommandargs 和 https://v2.nonebot.dev/docs/tutorial/plugin/create-handler#shellcommandargv 导入的函数是on_command,但是后面调用的是on_shell_command,应该是写错了,至于这个页面其他实例代码有没有类似的情况我没看,只看了on_command和on_shell_command的四个
